README.md

```diff
@@ -1,2 +1,45 @@
 # ranked-voting
-Preferential voting using ranked voting method
+## Preferential voting using ranked voting method
+---
+# setup
+
+## install python3
+
+## install pip3 (python package manager for python3)
+
+## recomended use venv (python virtual environment something like docker container for python project) doc bellow:
+> ## https://packaging.python.org/guides/installing-using-pip-and-virtual-environments/
+## to activate venv use: 
+> ## source venv/bin/activate (unix base)
+> ## .\env\Scripts\activate (windows)
```

Comment:
This should be bash code, right?
Using `>` means it's a quote, not a code. Did you try it with any markdown preview?
you should be using tripple `

Reply:
Oh... i forgot this command, ill change